template.yaml

```diff
@@ -0,0 +1,970 @@
+        - Key: Owner
+          Value: di-orchestration@digital.cabinet-office.gov.uk
+
+  ContainerService:
+    Type: AWS::ECS::Service
+    Properties:
+      Cluster: !Ref FargateCluster
+      TaskDefinition: !Ref TaskDefinition
+      DesiredCount: 1
+      LaunchType: FARGATE
+      NetworkConfiguration:
+        AwsvpcConfiguration:
+          AssignPublicIp: DISABLED
+          Subnets:
+            - Fn::ImportValue: !Sub ${VpcStackName}-ProtectedSubnetIdA
+            - Fn::ImportValue: !Sub ${VpcStackName}-ProtectedSubnetIdB
+          SecurityGroups:
+            - !Ref ContainerServiceSecurityGroup
+      LoadBalancers:
+        - ContainerName: !Sub "${AWS::StackName}-ECSServiceTaskDefinition"
+          ContainerPort: 8080
+          TargetGroupArn: !Ref ApplicationLoadBalancerTargetGroup
+      PropagateTags: SERVICE
+      Tags:
+        - Key: Name
+          Value: !Sub "${AWS::StackName}-ContainerService"
+        - Key: Service
+          Value: Relying Party stub
+        - Key: Source
+          Value: alphagov/di-auth-stub-relying-party
+        - Key: Owner
+          Value: di-orchestration@digital.cabinet-office.gov.uk
+    DependsOn: ApplicationLoadBalancerListener
+
+  ContainerServiceSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Security Group to access the Container Service
+      GroupName: !Join
+        - "-"
+        - - !Ref AWS::StackName
+          - ContainerService
+          - Fn::Select:
+              - 4
+              - Fn::Split:
+                  - "-"
+                  - Fn::Select:
+                      - 2
+                      - Fn::Split:
+                          - "/"
+                          - Ref: AWS::StackId
+      SecurityGroupIngress:
+        - Description: Allow traffic from the load balancer on port 8080
+          SourceSecurityGroupId: !GetAtt ApplicationLoadBalancerSecurityGroup.GroupId
+          IpProtocol: tcp
+          FromPort: 8080
+          ToPort: 8080
+      VpcId:
+        Fn::ImportValue: !Sub ${VpcStackName}-VpcId
+      Tags:
+        - Key: Name
+          Value: !Sub "${AWS::StackName}-ContainerServiceSecurityGroup"
+        - Key: Service
+          Value: Relying Party stub
+        - Key: Source
+          Value: alphagov/di-auth-stub-relying-party
+        - Key: Owner
+          Value: di-orchestration@digital.cabinet-office.gov.uk
+
+  DocAppContainerService:
+    Type: AWS::ECS::Service
+    Properties:
+      Cluster: !Ref FargateCluster
+      TaskDefinition: !Ref DocAppTaskDefinition
+      DesiredCount: 1
+      LaunchType: FARGATE
+      NetworkConfiguration:
+        AwsvpcConfiguration:
+          AssignPublicIp: DISABLED
+          Subnets:
+            - Fn::ImportValue: !Sub ${VpcStackName}-ProtectedSubnetIdA
+            - Fn::ImportValue: !Sub ${VpcStackName}-ProtectedSubnetIdB
+          SecurityGroups:
+            - !Ref DocAppContainerServiceSecurityGroup
```

Comment:
This one I'm going to double check with Ethan, just because we were going to not worry too much about duplication here in the aim to get something ready as quick as possible